base/src/main/proto/spine/options.proto

```diff
@@ -735,6 +747,16 @@ message GoesOption {
     string msg_format = 2;
 }
 
+// Defines options of a repeated field that defines a logic
+enum OnDuplicate {
```

Comment:
As discussed vocally, please use `distinct` boolean option so that we have true `Set` behavior.